contracts/resolvers/DefaultReverseResolver.sol

```diff
@@ -9,32 +9,30 @@ import "../registry/ReverseRegistrar.sol";
  */
 contract DefaultReverseResolver {
     // namehash('addr.reverse')
-    bytes32 constant ADDR_REVERSE_NODE = 0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2;
+    bytes32 constant ADDR_REVERSE_NODE =
+        0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2;
 
     ENS public ens;
-    mapping (bytes32 => string) public name;
+    mapping(bytes32 => string) public name;
+    address immutable owner;
+    event NameChanged(bytes32 indexed node, string name);
 
     /**
      * @dev Only permits calls by the reverse registrar.
      * @param node The node permission is required for.
      */
     modifier onlyOwner(bytes32 node) {
-        require(msg.sender == ens.owner(node));
+        require(msg.sender == owner);
         _;
     }
 
     /**
      * @dev Constructor
      * @param ensAddr The address of the ENS registry.
      */
-    constructor(ENS ensAddr) {
+    constructor(ENS ensAddr, address reverseRegistrar) {
         ens = ensAddr;
-
-        // Assign ownership of the reverse record to our deployer
-        ReverseRegistrar registrar = ReverseRegistrar(ens.owner(ADDR_REVERSE_NODE));
-        if (address(registrar) != address(0x0)) {
-            registrar.claim(msg.sender);
-        }
```

Comment:
Which test does it cover this change?

Reply:
Default reverse resolver isn't used anymore. Going to delete